fidesctl/src/fidesctl/core/export.py

```diff
@@ -0,0 +1,219 @@
+"""This module handles the logic required for applying manifest files to the server."""
+from datetime import datetime
+from typing import Dict, List, Tuple, Set
+
+import csv
+
+from fidesctl.core.api_helpers import get_server_resources
+from fidesctl.core.utils import echo_green, get_all_level_fields
+
+
+def list_fides_keys(resources: List) -> List:
```

Comment:
this seems like too small of a thing to be its own function, was there a case that necessitated it?

Reply:
Mostly just around the DRY aspect when planning for separate commands, I think I originally saw we were doing something similar in a couple of other places as well so considered shifting this into `fidesctl.core.utils` for a future refactor and consistency. Happy to get rid of it and build the list separately in each, definitely see what you mean about being to small to warrant its own function 👍🏽